.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,6 @@
+-   id: clj-kondo
+    name: clj-kondo
+    description: '`clj-kondo` is a command-line utility for enforcing style consistency across Clojure projects.'
+    entry: clj-kondo --lint
+    language: system
+    types: [clojure]
```

Comment:
Does this also cover clojurescript?

Reply:
Yes. It'll cover .Clj .cljs and .edn files.

Reply:
Thanks!